OpenRA.Mods.Common/Lint/CheckUnknownTraitFields.cs

```diff
@@ -37,7 +37,14 @@ void CheckActors(IEnumerable<MiniYamlNode> actors, Action<string> emitError, Mod
 					// Removals can never define children
 					if (t.Key.StartsWith("-", StringComparison.Ordinal) && t.Value.Nodes.Any())
 					{
-						emitError("{0} has child nodes, which is not valid for removals.".F(t.Key));
+						emitError("{0} defines child nodes, which are not valid for removals.".F(t.Location));
+						continue;
+					}
+
+					// Inherits can never define children
+					if (NormalizeName(t.Key) == "Inherits" && t.Value.Nodes.Any())
```

Comment:
It would make more sense to me if you moved `var traitName = NormalizeName(t.Key);` above this and used `traitName` instead of `NormalizeName` a second time.

Reply:
Done.